src/Equinor.ProCoSys.IPO.Command/InvitationCommands/InvitationHelper.cs

```diff
@@ -1,9 +1,52 @@
-﻿using Equinor.ProCoSys.IPO.Domain.AggregateModels.InvitationAggregate;
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using Equinor.ProCoSys.IPO.Domain.AggregateModels.InvitationAggregate;
+using Equinor.ProCoSys.IPO.ForeignApi;
+using Fusion.Integration.Meeting;
 
 namespace Equinor.ProCoSys.IPO.Command.InvitationCommands
 {
-    public class MeetingInvitationHelper
+    public class InvitationHelper
     {
+        public static List<BuilderParticipant> AddPersonToOutlookParticipantList(
+            ProCoSysPerson person,
+            List<BuilderParticipant> participants,
+            bool required = true)
+        {
+            if (required)
+            {
+                if (IsValidEmail(person.Email))
```

Comment:
@janid1967 @paal123 Email is required and has email verification in UI in old PCS. Should I still have a verification here in my code?

Reply:
Im not very fond of these static classes. But I would at least suggest to re-structure the method a bit to something like this:

```
public static List<BuilderParticipant> AddPersonToOutlookParticipantList(
            ProCoSysPerson person,
            List<BuilderParticipant> participants,
            bool required = true)
{
            var participantType = GetParticipantType(required);
            var participantIdentifier = CreateParticipantIdentifier(person);

            participants.Add(new BuilderParticipant(participantType, participantIdentifier));
            return participants;
}

private static ParticipantIdentifier CreateParticipantIdentifier(ProCoSysPerson person)
{
            if (IsValidEmail(person.Email))
            {
                return new ParticipantIdentifier(person.Email);
            }
            
            return new ParticipantIdentifier(new Guid(person.AzureOid));
}

private static ParticipantType GetParticipantType(bool required) => required ? ParticipantType.Required : ParticipantType.Optional;

```

Reply:
Okay.

Need to have the email check anyhow because of this requested change: https://statoildeveloper.visualstudio.com/Statoil.NextGenProCoSys/_workitems/edit/80984